src/js/legend.js

```diff
@@ -273,6 +273,7 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
         that.events.onLegendCreated.fire();
     };
 
+    // TODO: refactor into a utility function, now used by sonifier
```

Comment:
@cindyli Does it make sense to make a separate `chartAuthoringUtils.js` file? We've got `d3Utils.js`, but an ascending sort isn't D3-specific. 

Alternately, maybe `d3Utils.js` should become `chartAuthoringUtils.js`.